src/coreclr/vm/methodtable.h

```diff
@@ -2926,8 +2940,23 @@ public :
 
         virtual UINT32 GetNumVirtuals()
             { LIMITED_METHOD_CONTRACT; return m_pDeclMT->GetNumVirtuals(); }
+
+        static UINT32 ComputeNumMethods(MethodTable *pMT, MethodDataComputeOptions computeOptions)
+        {
```

Comment:
Contract? Also please assert `pMT` is non-null.

Reply:
Why assert? Its gonna AV immediately if pMT is NULL

Reply:
Asserts are much easier to capture in a debugger and integrate better when a debugger isn't attached.